### ISSUE TITLE
no-unused-variable false positive when let/const is declared in a require statement

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#504]
this code incorrectly reports that `fs` is unused:

```ts
let fs = require("fs"); // failure

fs.readFileSync("test.ts");
```

replacing `let` with `const` results in the same failure.

**Comment by leeavital** [comment]
Duplicate of #325, which is due to a problem in language services.

**Comment by adidahiya** [comment]
I just tried it out and this bug + #325 are fixed with TS v1.5.3
